nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -498,7 +498,7 @@ sub in {
     my $console_width = $1, my $console_height = $2;
     if ($console_width > 1920) {
         push @attrs, "# High-DPI console";
-        push @attrs, 'console.font = lib.mkDefault "${pkgs.terminus_font}/share/consolefonts/ter-u28n.psf.gz";';
+        push @attrs, 'console.font = lib.mkDefault "latarcyrheb-sun32";';
```

Comment:
How about enabling the high-dpi support option introduced in this PR ?

Reply:
will switch.